Sources/Messages/BadgeMessage/BadgeMessageView.swift

```diff
@@ -237,13 +238,32 @@ open class BadgeMessageView: UIView {
     
     public override func layoutSubviews() {
         super.layoutSubviews()
-        updateLayer()
+        updateCornersAndShadow()
     }
     
     /// Update cornerRadius and shadow
-    private func updateLayer() {
-        updateCornerRadius(Constants.cornerRadius)
-        updateRoundedShadowPath()
+    private func updateCornersAndShadow() {
+        let layers = [layer, containerView.layer, shadowView.layer]
+        
+        // cornerRadius
+        layers.forEach {
+            $0.updateCornerRadius(Constants.cornerRadius)
+        }
+        
+        if #available(iOS 13.0, *) {
+            layers.forEach {
+                $0.cornerCurve = .continuous
+            }
+        }
+        
+        shadowView.layer.applySketchShadow(
```

Comment:
Does this not break being able to control this shadow externally to this package? :/ Could we perhaps not extend `ShadowComponents` (to include spread) and then have a `get` var on it to convert to a sketch shadow?

Reply:
@simonmitchell It does indeed, as it's kinda a fixed design and we weren't overriding it I feel that was too much of an issue. 

But agreed, a configurable `shadowComponents`  property which we could convert to a `SketchShadow` `struct` would be a really nice solution. Just need to decide if we add properties to `ShadowComponents` 🤔

Reply:
@simonmitchell Addressed in PR #21